### PR TITLE
V0.0 kevin - final changes

### PR DESCRIPTION
**Pull request by kevinLamKB**
Update UserGuide

### CODE REVIEW

**Comment by kevinLamKB** [comment]
Added DeveloperGuide

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -511,70 +503,66 @@ Use case ends.
...
 ii) User specifies index.
-iii) Use case resumes from step 3.
+iii) Use case resumes from step 2.
 
 [appendix]
 == Non Functional Requirements
 
 .  The software should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
-.  The app should be able to hold up to 1000 entries without a noticeable sluggishness in performance for typical usage.
+. The app should be able to hold up to 1000 entries without a noticeable sluggishness in performence for typical usage.
```
Comment (mattheuslee) [inline docs/DeveloperGuide.adoc]: typo?

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -511,70 +503,66 @@ Use case ends.
...
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 .  The software should use text files for storage and not relational databases.
-.  The software should favor DOS style commands over Unix-style commands.
+.  The software should favor DOS stile commands over Unix-style commands.
```
Comment (mattheuslee) [inline docs/DeveloperGuide.adoc]: typo?

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -278,19 +278,22 @@ An entry *can* have:
...
 
-[[Event]]
+[[event]]
+Event
 ____
 An entry in the Task Manager with
 * Specified start *and* end date and/or time
 ____
 
 [[Deadline]]
```
Comment (ChuaPingChan) [inline docs/UserGuide.adoc]: +[[deadline]]

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (kevinLamKB) [reply docs/DeveloperGuide.adoc]: resolved, thanks

File: docs/UserGuide.adoc (hunk 3 above)
Comment (kevinLamKB) [reply docs/UserGuide.adoc]: fixed, thanks

**Review by mattheuslee — APPROVED** [review]

File: docs/UserGuide.adoc (hunk 3 above)
Comment (ChuaPingChan) [inline docs/UserGuide.adoc]: @kevinLamKB 
Thanks for the detailed vetting.

**Review by ChuaPingChan — APPROVED** [review]
Thanks for the detailed vetting.

**Comment by mattheuslee** [comment]
@akd13 okay to merge?

**Review by akd13 — APPROVED** [review]

**Pull request merged**